base/multidimensional.jl

```diff
@@ -529,6 +529,12 @@ function accumulate_pairwise{T}(op, v::AbstractVector{T})
     return accumulate_pairwise!(op, out, v)
 end
 
+"""
+    cumsum!(B, A, [dim])
+
+Cumulative sum of `A` along a dimension, storing the result in `B`. The dimension defaults
+to 1.
+"""
```

Comment:
probably should attach this to the more general method `cumsum!(B, A, axis::Integer=1) = accumulate!(+, B, A, axis)` below instead of one specific to `v::AbstractVector`?